src/sensor/ping360.h

```diff
@@ -274,15 +364,19 @@ class Ping360 : public PingSensor
     uint16_t _angle = 0;
     uint16_t _transmit_duration = 42;
     uint32_t _ping_number = 0;
-    uint32_t _scan_start = 0;
-    uint32_t _scan_length = 0;
     uint32_t _gain_setting = 1;
     uint16_t _sample_period = 1300;
     uint32_t _speed_of_sound = 1500;
     uint16_t _transmit_frequency = 500;
 ///@}
 
-    uint16_t _num_points = 2000;
+    int _angle_offset = 0;
+    int _angular_speed = 1;
+    uint _central_angle = 1;
+    uint16_t _num_points = 1000;
+    bool _reverse_direction = false;
```

Comment:
My preference is to call this variable _direction, and handle it as the inverse, so you don't have to apply that double-negative logic in your head when you read the code.

If we want to be really c++ with it and clear we could have an
```
enum ScanDirection {
  NORMAL,
  REVERSE,
}
```

Reply:
doesn't hold pr.

Reply:
I think we should do that in the entire pingviewer. Will add an issue for that.